catboost/libs/metrics/metric.cpp

```diff
@@ -544,6 +545,84 @@ TVector<TParamSet> TMultiRMSEMetric::ValidParamSets() {
     return {TParamSet{{TParamInfo{"use_weights", false, true}}, ""}};
 };
 
+/* MultiRMSEWithMissingValues */
+namespace {
+    struct TMultiRMSEWithMissingValues final: public TAdditiveMultiRegressionMetric {
+        explicit TMultiRMSEWithMissingValues(const TLossParams& params)
+           : TAdditiveMultiRegressionMetric(ELossFunction::MultiRMSE, params)
```

Comment:
ELossFunction::MultiRMSEWithMissingValues

Reply:
right... thank you @LyzhinIvan 
Done.